packages/map-template/src/components/Notification/Notification.scss

```diff
@@ -0,0 +1,4 @@
+.notification {
+    z-index: 10000000;
```

Comment:
Is the z-index set a level higher than the splash screen in order to be the most visible element?